content/boilerplates/notes/1.1.12.md

```diff
@@ -0,0 +1,3 @@
+## Bug fixes
+
+- Fix a bug where the sidecar could infinitely forward requests to itself when pod defines a port that isn't defined for a service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
```

Comment:
when a `Pod` resource defines

Reply:
```suggestion
- Fix a bug where the sidecar could infinitely forward requests to itself when a `Pod` resource defines a port that isn't defined for a service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
```

Reply:
@geeknoid turning that into a suggestion makes it much easier on the author ^